recipes/doxygen/all/conanfile.py

```diff
@@ -1,11 +1,11 @@
 from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, rm, replace_in_file
 from conan.tools.scm import Version
-from conan import tools
 from conans import CMake
```

Comment:
Hmm 🤔 it's very odd you are using the old helper but deleted the cmake wrapper. That's required to add the Conan settings in correctly.

Was this intentional?